app/components/Dashboard/programAdministration.tsx

```diff
@@ -31,21 +31,26 @@ const ProgramDataManagement: React.FunctionComponent<Props> = ({
   return (
     <Card className="admin-control-card">
       <Card.Body>
-        <Card.Title>Program and Data management</Card.Title>
+        <Card.Title>Program Administration</Card.Title>
         <Card.Text>
-          Update program parameters, view program insights etc.
+          Review applications and administer program details:
         </Card.Text>
       </Card.Body>
       <ListGroup variant="flush">
-        {EditableItems}
+        <ListGroup.Item>
+          <Link href="/analyst/applications">
```

Comment:
I think this is one of these cases where you want to use the `passHref` prop: https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

Reply:
Interesting, I never noticed that before -

> Without this, the <a> tag will not have the href attribute, which might hurt your site’s SEO.

...not to mention the ability to right click and open the correct link in a new tab. Will fix.